<launch>

  <include file="$(find realsense2_camera)/launch/rs_camera.launch">
	<arg name="camera"              value="camera1"/>
	<arg name="serial_no"           value="948122071492"/>
	<arg name="color_fps"           value="15"/>
	<arg name="enable_gyro"         value="false"/>
	<arg name="enable_accel"        value="false"/>
  </include>

  <include file="$(find realsense2_camera)/launch/rs_camera.launch">
	<arg name="camera"              value="camera2"/>
	<arg name="serial_no"           value="949122070603"/>
	<arg name="color_fps"           value="15"/>
	<arg name="enable_gyro"         value="false"/>
	<arg name="enable_accel"        value="false"/>
  </include>

	<!--<param name ="/use_sim_time" value="true"/>-->
<param name ="/use_sim_time" value="false"/>
    <node ns="node1" pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager"/>
    <node ns="node2" pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager"/>  

    <node ns="node1" pkg="nodelet" clear_params="true" type="nodelet" name="aruco_detect" args="load aruco_pose/aruco_detect nodelet_manager" >
        <remap from="image_raw" to="/camera1/color/image_raw"/>
        <remap from="camera_info" to="/camera1/color/camera_info"/>
        <param name="length" value="0.33"/>
    </node>

    <node ns="node2" pkg="nodelet" clear_params="true" type="nodelet" name="aruco_detect" args="load aruco_pose/aruco_detect nodelet_manager">
        <remap from="image_raw" to="/camera2/color/image_raw"/>
        <remap from="camera_info" to="/camera2/color/camera_info"/>
        <param name="length" value="0.33"/>
    </node>


</launch>
